pyproject.toml

```diff
@@ -40,5 +41,8 @@ dependencies = [
     "pyusb==1.3.1",
     "requests>=2.32.4",
     "sh==1.4",
+    "sphinx~=8.2",
+    "sphinx-book-theme~=1.1",
+    "sphinx-design~=0.6",
```

Comment:
docs dependencies should leave in its own file/project, not everybody will want to build docs locally.

Reply:
I guess that's what we want yes, looks like we could do `pip install --group docs`